test/integration/ink_editor_spec.mjs

```diff
@@ -111,8 +111,6 @@ describe("Ink Editor", () => {
     it("must draw, undo/redo and check that the editor don't move", async () => {
       await Promise.all(
         pages.map(async ([browserName, page]) => {
-          await clearAll(page);
```

Comment:
Note that this is no longer necessary anymore as it was only here to remove ink annotations created by the previous test, but now that we close the document between tests the document is always empty already when this test runs as we don't inherit state from the previous test anymore.